lfucg-exactions/server/plats/views.py

```diff
@@ -22,161 +139,214 @@ def get(self, request, *args, **kwargs):
             'Unit',
             'Block',
             'Section',
-            'Cabinet',
-            'Slide Number',
             'Non-Sewer Exactions',
             'Sewer Exactions',
+            'Account',
         ]
 
-        plat = request.GET.get('plat')
+        all_rows = []
 
-        plat_queryset = Plat.objects.filter(id=plat)
+        plat_value = request.GET.get('plat', None)
 
-        if plat_queryset is not None:
-            plat_object = plat_queryset[0]
+        if plat_value is not None:
+            plat_queryset = Plat.objects.filter(id=plat_value)
+            plat_serializer = self.list(
+                plat_queryset,
+                PlatSerializer,
+                many=False
+            )
+            filename = plat_queryset[0].cabinet + '-' + plat_queryset[0].slide + '_plat_report.csv'
```

Comment:
I think this is okay, but want to double-check.  Am I right to assume there is no way for plat_queryset to be empty?  If it could be empty, then we will need a guard against this [0] index.

Reply:
Above this, I am getting the plat_value (and setting to None if it is not there).  I'm checking if plat_value is not None before getting the queryset.  Does this prevent problems?